docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>For clarity, I would add one additional step in between these two equations that takes the <code>m + k</code> summation and splits them into two separate summations (both staying under the square root) that go from <code>t = 1</code>  to <code>t = m</code>  and then <code>t = m + 1</code> to <code>t = m + k</code> . Then, it becomes obvious why the "greater than or equal to" must be <code>True</code> (i.e., if we remove the second summation which must be greater or equal to zero, then the bottom equation becomes a lower bound, so to speak)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>